test/cli/test_cli_functions.sh

```diff
@@ -119,7 +119,15 @@ function timeout {
 }
 
 function install {
-    test_noobaa install --mini
+    use_obc_cleanup_flag=$((RANDOM%2))
+    
+    if [ ${use_obc_cleanup_flag} -gt 0 ] 
+    then
+        test_noobaa install --mini --use-obc-cleanup-policy
+    else      
+        test_noobaa install --mini
+    fi  
```

Comment:
```suggestion
    local use-obc-cleanup-policy
    
   [ $((RANDOM%2)) -gt 0 ] && use-obc-cleanup-policy="--use-obc-cleanup-policy"
   test_noobaa install --mini ${use-obc-cleanup-policy}
```
Not sure how readable this suggestion, what do you think?

Reply:
I think it's great :) i'll change it